vars/satellite_common.yml

```diff
@@ -0,0 +1,27 @@
+---
+rhsm_username: "user@redhat.com"
+rhsm_password: "updatethisvalue"
+
+# Pass the pool-id which would have all the below repos available.
+#
+# Below for example:
+# Satellite Employee Subscriptions brings in all below repos.
+
+# "rhel-7-server-rpms"
+# "rhel-7-server-satellite-maintenance-6-rpms"
+# "rhel-server-rhscl-7-rpms"
+# "rhel-7-server-satellite-6.x-rpms"
+
+# Employee SKU brings in the below repos.
+# "rhel-7-server-ansible-2-rpms"
+
+# Satellite Employee subscriptions for all other Product/Repos.
+rhsm_pool_id1: 8a85f981519abf020151a22d3c387f2a
+# Employee SKU for 'Red Hat Ansible Engine Product'.
+rhsm_pool_id2: 8a85f98960dbf6510160df23e3367451
+
+# Dogfood URL
+dogfood_url: "http://dogfood.example.com"
```

Comment:
Per earlier comments, I'd change this just to be not specific to dogfood.

Reply:
or as per the best-practices.